playbooks/ansible-network-nxos-appliance/files/bootstrap.yaml

```diff
@@ -4,8 +4,8 @@
 - hosts: appliance
   gather_facts: false
   vars:
-    ansible_password: admin
     ansible_ssh_user: admin
+    ansible_password: qXq6vQPmU28U
```

Comment:
Can you use `ansible_ssh_pass` instead? I vaguely remember hitting an issue where _password doesn't work.

Reply:
Done